### PR TITLE
Document files in theorems/{homotopy,cohomology}/

### PR DESCRIPTION
**Pull request by timjb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -379,4 +366,5 @@ The views and conclusions contained in this document are those of the author and
 interpreted as representing the official policies, either expressed or implied, of any sponsoring
 institution, the U.S. government or any other entity.
 
-[favonia-thesis]: https://www.math.ias.edu/~favonia/thesis.html
\ No newline at end of file
+[favonia-thesis]: https://www.math.ias.edu/~favonia/thesis.html
```
Comment (favonia) [inline README.md]: Thanks. I would suggest http://favonia.org/thesis.html till I have a permanent job somewhere. :sweat_smile:

File: README.md [hunk 2]
```diff
@@ -245,77 +245,64 @@ TODO: describe more precisely each file
...
+- `HopfConstruction`: Given a 0-connected H-space `X`, constructs a fibration `H` on `Susp A` with total space equivalent to the join `X * X`.
+- `HopfJunior`: Contains `HopfJunior : S¹ → Type₀`, a fibration with fibers equivalent to `Bool` (a.k.a. the 0-sphere) and a proof that its total space is (equivalent to) `S¹`.
+- `IterSuspensionStable`: Contains a reformulation of the Freudenthal suspension theorem.
+- `JoinAssoc3x3`: Gives an equivalence between the joins `(A * B) * C` and `A * (B * C)`. The proof uses the 3x3-lemma.
+- `JoinAssocCubical`: Gives an equivalence between the joins `(A * B) * C` and `A * (B * C)`. The proof involves squares and cubes.
+- `JoinComm`: Gives an equivalence between the joins `A * B` and `B * A`.
+- `JoinSusp`: Contains equivalences `Bool * A ≃ Susp A`, `Susp A * B ≃ Susp (A * B)` and `⊙Sphere m ⊙* X ⊙≃ ⊙Susp^ (S m) X` ((m+1)-fold suspension is equivalent to joining with an m-sphere).
+- `LoopSpaceCircle`: Proves that the fundamental group of the circle is equivalent to the integers.
+- `ModalWedgeExtension`: Lemmas about modalities and the function `X ∨ Y → X × Y` for pointed types `X` and `Y`.
```
Comment (favonia) [inline README.md]: FYI: This will play the same role as `WedgeExtension` in the generalized Blakers-Massey theorem.

**Comment by favonia** [comment]
This looks great. Thanks! Just signal me when you think this is ready.

**Comment by timjb** [comment]
@favonia I've also added descriptions of files in `theorems/cohomology/`. I think this is now ready to be merged.

File: README.md [hunk 3]
```diff
@@ -306,9 +306,30 @@ This directory contains proofs of interesting homotopy-theoretic theorems.
...
+- `Coproduct`: Proves that `C n (X ⊙⊔ Y) ≃ᴳ C n (X ⊙∨ Y) ×ᴳ C2 n` (where `C2 n` is the `n`-th cohomology of the 0-sphere) for any cohomology theory `C`.
+- `DisjointlyPointedSet`: Shows that the cohomology of a separable pointed set `X`, which has choice, is the `MinusPoint X`-fold product of `C2 0` (the 0-th cohomology of the 0-sphere) in degree 0 and trivial in higher degrees for any ordinary cohomology theory `C`.
+- `EMModel`: Constructs the Eilenberg–MacLane spectrum given an abelian group and shows that its induced cohomology theory is ordinary.
+- `InverseInSusp`: Shows that the homomorphism Cⁿ(ΣX) → Cⁿ(ΣX) mapping an element to its inverse is induced by a map ΣX → ΣX.
+- `LongExactSequence`: Given a map `f : X → Y`, constructs the sequence Cⁿ(Y) → Cⁿ(X) → Cⁿ⁺¹(cofib(f)) → Cⁿ⁺¹(Y) → ⋯ and shows that it is exact.
+- `MayerVietoris`: Given a pointed span X ←f– Z –g→ Y, shows the cofiber space of the natural map `reglue` : X ∨ Y → X ⊔_Z Y is equivalent to the suspension of Z. Using this equivalence one can derive the Mayer–Vietoris sequence from the long exact sequence associated with `reglue`.
```
Comment (favonia) [inline README.md]: Hmm. Somehow GitHub-markdown is unhappy about the underscore in `⊔_Z`. Could you escape it?

Comment (timjb) [reply README.md]: fixed

**Comment by favonia** [comment]
Thanks for your contribution. Maybe we should switch to reStructuredText for citations...

**Pull request merged**